### PR TITLE
Update docker start procedure (Draft in progress)

### PR DESCRIPTION
**Pull request by keegoid-nr**
- Docker Engine must be installed first
- The Linux Terminal must be used for all platforms (including Windows)
- Clarify common permissions issues

### CODE REVIEW

**Comment by keegoid-nr** [comment]
Requesting review from @bpeckNR.

**Review by mderemer-nr — APPROVED** [review]

**Comment by paperclypse** [comment]
@keegoid-nr Are we still waiting for a review from @bpeckNR on this one?

**Comment by austin-schaefer** [comment]
Closing out all open 2022 PRs, feel free to re-open if there's active work here or this becomes active again!

**Pull request closed without merging**
